modules/library/common-search/src/main/java/org/fao/geonet/common/search/GnMediaType.java

```diff
@@ -55,5 +58,6 @@ public GnMediaType(String type) {
     APPLICATION_GN_XML = new MediaType("application", "gn+xml");
     APPLICATION_JSON_LD = new MediaType("application", "ld+json");
     APPLICATION_GEOJSON = new MediaType("application", "geo+json");
+    APPLICATION_ELASTICJSON = new MediaType("application", "elastic+json");
```

Comment:
```suggestion
    APPLICATION_ELASTICJSON = new MediaType("application", "gnindex+json");
```
maybe?

Reply:
thanks @fxprunayre - I've changed it to "gnindex+json".

Thanks for the review - its really really helpful!